generation/um/WinTrust/generate.rsp

```diff
@@ -0,0 +1,73 @@
+--additional
+-m32
+--config
+multi-file
+--file
+um-WinTrust.h
+--headerFile
+header.txt
+--include-directory
+C:/Program Files (x86)/Windows Kits/10/Include/10.0.18362.0/shared
+--include-directory
+C:/Program Files (x86)/Windows Kits/10/Include/10.0.18362.0/um
+--libraryPath
+Wintrust
```

Comment:
nit: This should be lowercase to match the actual filename `wintrust.dll`

Reply:
Fixed.